src/backends/torch/torchutils.h

```diff
@@ -35,6 +35,8 @@
 #pragma GCC diagnostic pop
 #include <torch/script.h>
 
+#include <google/protobuf/message.h>
```

Comment:
why this?

Reply:
do not know why it triggered here:  in case of separate compilation I got sometimes a "google::protobuf::Message" not declared ...

Reply:
Strange... Where is the error located? It may be in a file that includes a file that itself includes torchutils.h or something like that

Reply:
yes, from something-json-something if I remember correctly

Reply:
but anywat as google::protobuf::message is used just below, I think it is a good idea to include it here :)